src/pandablocks_ioc/_pvi.py

```diff
@@ -206,22 +209,19 @@ def create_pvi_records(record_prefix: str):
         # Create top level Device, with references to all child Devices
         device_refs = [DeviceRef(x, x) for x in pvi_records]
 
-        # # TODO: What should the label be?
         device = Device("TOP", device_refs)
         devices.append(device)
 
         # TODO: label widths need some tweaking - some are pretty long right now
+        # TODO: Need to decide how to handle already existing directory/files.
+        # Could still be left over stuff from a previous run?
```

Comment:
It may be worth making a decision about this before merging and releasing. We wouldn't really want to change its behaviour after release, as it'd likely be a breaking change.